src/SFA.DAS.AANHub.Application.UnitTests/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Apprentices.Commands.PatchApprenticeMember
+{
+    [TestFixture]
+    public class PatchApprenticeMemberCommandTests
+    {
+        private const string Replace = "replace";
+        private const string Email = "Email";
+        private const string Name = "Name";
+
+        [Test]
+        public void Command_PatchContainsEmail_EmailIsSet()
+        {
+            var memberId = Guid.NewGuid();
+            var apprenticeid = 123;
```

Comment:
```suggestion
            var apprenticeId = 123;
```

Reply:
Done. Updated variable case in other classes too.